ignite/utils.py

```diff
@@ -271,3 +272,70 @@ def wrapper(*args: Any, **kwargs: Dict[str, Any]) -> Callable:
         return cast(F, wrapper)
 
     return decorator
+
+
+def expand_hostlist(nodelist: str) -> List[str]:
```

Comment:
I'm not sure if we need this in `ignite.utils` and as a public method. Let's keep it near in `ignite/distributed/comp_models/native.py` and as a private method...

Reply:
Ok 👍🏻